src/pages/index.js

```diff
@@ -163,34 +229,109 @@ function closeModalOnOverlayClick(modal) {
 //Saves any profile changes once the user edits the profile
```

Comment:
* The `confirmation` modal doesn't close on `overlay` clicks


Please, add it into the array of modals


```js

// Add overlay click functionality for each modal
[editModal, cardModal, previewModal, deleteModal].forEach((modal) => {
  closeModalOnOverlayClick(modal);
});
```

Reply:
One remark hasn't been corrected https://github.com/CODEM0N3T/se_project_spots/pull/1#discussion_r2036145378 


* The `confirmation` modal doesn't close on `overlay` clicks


Please, add `deleteModal` into the array of modals in line 225


```js

// Add overlay click functionality for each modal
[editModal, cardModal, previewModal, deleteModal].forEach((modal) => {
  closeModalOnOverlayClick(modal);
});
```

Reply:
* And `avatar` modal doesn't close when I click on the `overlay`. Please, add  `avatarModal` also into the array of modals in line 225


```js

// Add overlay click functionality for each modal
[editModal, cardModal, previewModal, deleteModal, avatarModal].forEach((modal) => {
  closeModalOnOverlayClick(modal);
});
```